main.go

```diff
@@ -220,40 +220,31 @@ func apply(configFile string) (ctrl.Options, config.Configuration) {
 		Scheme: scheme,
 	}
 
-	config := config.Configuration{}
+	cfg := config.Configuration{}
 
 	if configFile == "" {
-		scheme.Default(&config)
+		scheme.Default(&cfg)
+		options = ctrl.Options{
+			Scheme:                 scheme,
+			Port:                   config.DefaultWebhookPort,
+			HealthProbeBindAddress: config.DefaultHealthProbeBindAddress,
+			MetricsBindAddress:     config.DefaultMetricsBindAddress,
+			LeaderElection:         false,
+		}
```

Comment:
I wonder if this works
```suggestion
		options, err = options.AndFrom(cfg)
```

Reply:
That works fine on my local. Thanks!